integration/firestore/gulpfile.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+const gulp = require('gulp');
+const replace = require('gulp-replace');
+const { resolve } = require('path');
+const webpackStream = require('webpack-stream');
+const webpack = require('webpack');
+const filter = require('gulp-filter');
+
+function reworkFirebasePaths() {
+  return gulp
+    .src(
+      [
+        resolve(__dirname, '../../packages/firestore/**/*.ts'),
+        `!${resolve(__dirname, '../../packages/firestore/node_modules/**/*')}`,
+        `!${resolve(__dirname, '../../packages/firestore/dist/**/*')}`
+      ],
+      { base: '../../packages/firestore' }
+    )
+    .pipe(
+      replace(
+        /**
+      * This regex is designed to match the following statement used in our
```

Comment:
I guess prettier is messing up the formatting on these comments? :-/

Reply:
Sad day. I'll go fix it.